apps/src/studio/StudioAnimation.js

```diff
@@ -84,13 +95,32 @@ StudioAnimation.prototype.getElement = function () {
 
 /**
  * Create an image element with a clip path
+ * (if the element and clip path already exist, update the dimensions and URL)
  */
 StudioAnimation.prototype.createElement = function (parentElement) {
```

Comment:
Should this method be renamed, since it's now got a code path that doesn't create an element?